### PR TITLE
Make create job API to return more information

### PR DESCRIPTION
**Pull request by noorul**
POST /jobs/ should return as many information as GET /jobs/<jobId>

Also the returned json should not have a result key in the json to make
it consistent.

### CODE REVIEW

**Comment by hntd187** [comment]
I like your choice of classpath class. Well memed sir. LGTM

File: job-server/src/spark.jobserver/WebApi.scala [hunk 1]
```diff
@@ -86,17 +86,18 @@ object WebApi {
         "stack" -> t.getStackTrace.map(_.toString).toSeq)
     }
 
-  def getJobReport(jobInfo: JobInfo): Map[String, Any] = {
-    Map("jobId" -> jobInfo.jobId,
-      "startTime" -> jobInfo.startTime.toString(),
-      "classPath" -> jobInfo.classPath,
-      "context" -> (if (jobInfo.contextName.isEmpty) "<<ad-hoc>>" else jobInfo.contextName),
-      "duration" -> getJobDurationString(jobInfo)) ++ (jobInfo match {
+  def getJobReport(jobInfo: JobInfo, jobStarted:Boolean = false): Map[String, Any] = {
```
Comment (velvia) [inline job-server/src/spark.jobserver/WebApi.scala]: (style) space before Boolean

**Comment by velvia** [comment]
👍 except for one minor style thing.... thanks @noorul

**Comment by noorul** [comment]
@hntd187 It was already there :)

**Pull request merged**